### PR TITLE
Multiple integration bugfixes

### PR DESCRIPTION
**Pull request by nihco2**
| Questions | Answers |
| --- | --- |
| Branch? | develop |
| Description? | multiple bugfixes |
| Type? | bug fix |
| Category? | CO |
| BC breaks? | no |
| Deprecations? | no |
| Fixed ticket? | filtered bug list here: http://bit.ly/29sWf3I |
| How to test? | Check the bug list link above |

### CODE REVIEW

File: src/PrestaShopBundle/Form/Admin/Product/ProductQuantity.php [hunk 1]
```diff
@@ -124,7 +124,7 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 'options' => array(),
                 'locales' => $this->locales,
                 'hideTabs' => true,
-                'label' => $this->translator->trans('Label when out of stock', array(), 'Admin.Catalog.Feature'),
+                'label' => $this->translator->trans('Displayed text when out of stock', array(), 'Admin.Catalog.Feature'),
```
Comment (mickaelandrieu) [inline src/PrestaShopBundle/Form/Admin/Product/ProductQuantity.php]: ping @AlexEven (just in case)

**Comment by vincentbz** [comment]
Status : 'QA approved'
Status : 'PM approved'

Cool thanks :) All the tickets have been tested and closed

**Comment by vincentbz** [comment]
Status : 'PM approved'

File: src/PrestaShopBundle/Form/Admin/Product/ProductQuantity.php (hunk 1 above)
Comment (AlexEven) [inline src/PrestaShopBundle/Form/Admin/Product/ProductQuantity.php]: Thanks! I updated it in another PR https://github.com/PrestaShop/PrestaShop/pull/5901

**Pull request merged**